clients/go/pkg/zbc/client_test.go

```diff
@@ -421,8 +487,16 @@ func (s *clientTestSuite) TestOverrideHostAndPortEnvVar() {
 	s.EqualValues(fmt.Sprintf("%s:%s", address, port), config.GatewayAddress)
 }
 
-func createSecureServer() (net.Listener, *grpc.Server) {
-	creds, _ := credentials.NewServerTLSFromFile("testdata/chain.cert.pem", "testdata/private.key.pem")
+func createSecureServer(withSan bool) (net.Listener, *grpc.Server) {
+	certFile := "testdata/chain.cert.pem"
+	keyFile := "testdata/private.key.pem"
+
+	if withSan {
+		certFile = "testdata/chain.cert.san.pem"
+		keyFile = "testdata/private.key.san.pem"
+	}
```

Comment:
🔧 In the future someone might wonder what the difference between the `chain.cert.pem` and `chain.cert.san.pem` is. For this reason it might be useful to add to the commit message how this cert was created. A comment hightlighting the intent (or differences from the other cert) might also be useful.
 
Inspecting the certs with step (e.g. `step certificate inspect testdata/chain.cert.san.pem`) highlights that there are more differences than just the SAN.

Reply:
I updated the first commit and the PR description.